packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenuPage.tsx

```diff
@@ -3,14 +3,16 @@ import {
   ExampleCard,
   IComponentDemoPageProps,
   ComponentPage,
-  PropertiesTableSet
+  PropertiesTableSet,
+  ComponentStatus
 } from '@uifabric/example-app-base';
 import { ContextualMenuBasicExample } from './examples/ContextualMenu.Basic.Example';
 import { ContextualMenuSectionExample } from './examples/ContextualMenu.Section.Example';
 import { ContextualMenuCheckmarksExample } from './examples/ContextualMenu.Checkmarks.Example';
 import { ContextualMenuDirectionalExample } from './examples/ContextualMenu.Directional.Example';
 import { ContextualMenuCustomizationExample } from './examples/ContextualMenu.Customization.Example';
 import { ContextualMenuHeaderExample } from './examples/ContextualMenu.Header.Example';
+import { ComponentStatusState } from '../ComponentStatus/ComponentStatusState';
```

Comment:
After moving the ComponentStatus (and ComponentStatusState) stuff to experimental, all the imports on all of the components need to be updated

Reply:
talked with David... and I moved the ComponentStatus to demo folder as it will not be shipped.